DimaDK24/orders-manager-frontend
# Feat: add date pickers

[Trello ticket](https://trello.com/c/0JnHQQII)

# Problem statement

Make user possible to choose order date and delivery date

# Steps to test the changes

1) Start server by 'yarn start' command.
2) Click to date inputs and choose any date (and time for the order input)
3) See result

# Solution description

1) Create component called LabelledDateInput, which can take some props depend of requirements.
2) Use this component

# Sanity checklist


# Review checklist
